### PR TITLE
Update Contact Controller, Route and Template #49

### PR DESCRIPTION
**Pull request by zoltan-nz**

### CODE REVIEW

File: app/templates/contact.hbs [hunk 1]
```diff
@@ -13,14 +13,26 @@
...
+          <span class="glyphicon glyphicon-ok form-control-feedback"></span>
+        {{/if}}
       </div>
       <div class="form-group has-feedback {{if this.model.isMessageEnoughLong "has-success"}}">
         <label>Your message*:</label>
-        <Textarea class="form-control" placeholder="Your message. (At least 5 characters.)" @rows="5" @value={{this.model.message}} />        {{#if this.model.isMessageEnoughLong}}<span class="glyphicon glyphicon-ok form-control-feedback"></span>{{/if}}
+        <Textarea class="form-control" placeholder="Your message. (At least 5 characters.)" @rows="5"
+                  @value={{this.model.message}} />
+
+        {{#if this.model.isMessageEnoughLong}}
```
Comment (tkraak) [inline app/templates/contact.hbs]: How about we rename this model property to `isMessageLongEnough`?

File: tests/unit/controllers/contact-test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+
+const { stub } = sinon;
+
+module('Unit | Controller | contact', function(hooks) {
+  setupTest(hooks);
+
+  test('sendMessage action', function(assert) {
+    let controller = this.owner.lookup('controller:contact');
```
Comment (tkraak) [inline tests/unit/controllers/contact-test.js]: Why `let` instead of `const` here?

File: tests/unit/controllers/contact-test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    const save = stub().returns({ then: stub().yields() });
+    const newContactMessage = { save };
+    controller.send('sendMessage', newContactMessage);
```
Comment (tkraak) [inline tests/unit/controllers/contact-test.js]:
```js
controller.sendMessage(newContactMessage);
```

File: app/templates/contact.hbs (hunk 1 above)
Comment (zoltan-nz) [reply app/templates/contact.hbs]: Good call. Thanks. Done.

File: tests/unit/controllers/contact-test.js (hunk 3 above)
Comment (zoltan-nz) [reply tests/unit/controllers/contact-test.js]: Ah yep. Thank you. Done.

File: tests/unit/controllers/contact-test.js (hunk 2 above)
Comment (zoltan-nz) [reply tests/unit/controllers/contact-test.js]: Yep. Totally agree. Changed. Thanks. :)

**Pull request merged**